packages/x-components/src/x-modules/search/store/actions/save-search-response.action.ts

```diff
@@ -23,6 +23,19 @@ export const saveSearchResponse: SearchXStoreModule['actions']['saveSearchRespon
     queryTagging
   }
 ) => {
+  if (state.isNoResultsWithFilters && state.results.length) {
+    commit('setIsNoResultsWithFilters', false);
+  }
```

Comment:
What do you think of moving this block to the request so the `isNoResultsWithFilters` flag is reset right when the request is made?